docs/azure.md

```diff
@@ -0,0 +1,268 @@
+# AWS
```

Comment:
`s/AWS/Azure/g`

Reply:
@colemickens Ya, still working the docs. This is just copy pasta for now.